README.md

```diff
@@ -33,7 +33,11 @@ market.
 This is not a good example for code quality or structure, many LOC are written
 while fighting or other occasions which needed quick fixes or in the ingame
 editor. But I think there are a couple of funny ideas. Every contribution is
-welcome. 
+welcome.
+
+Copy `src/config_local.js.example` to `src/config_local.js` and use it to
```

Comment:
We could put the `config_local.js.example` in the root. So that in the `src/` directory are only `.js` files. No strong opinion on this, just an idea.